packages/airnode-examples/src/scripts/deploy-rrp-dry-run.ts

```diff
@@ -0,0 +1,8 @@
+import { cliPrint, deployContract, runAndHandleErrors } from '../';
+
+const main = async () => {
+  const airnodeRrpDryRun = await deployContract('@api3/airnode-protocol/contracts/rrp/AirnodeRrpV0DryRun.sol');
+  cliPrint.info(`AirnodeRrpV0DryRun deployed to address: ${airnodeRrpDryRun.address}`);
```

Comment:
Nit: In the `packages/airnode-examples/src/scripts/deploy-rrp.ts`, we print the message `AirnodeRrp deployed to address: ${airnodeRrp.address}` which doesn't include `V0`. We can either change deploy-rrp.ts to message `AirnodeRrpV0 ...` or deploy-rrp-dry-run.ts to message `AirnodeRrpDryRun ...`.

Reply:
Nice catch. I like both having `V0` since both contract names have `V0`.